Sources/SwiftRouting/Router/RouterContext.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  RouterContext.swift
+//  swift-routing
+//
+//  Created by Kévin Budain on 3/3/25.
+//
+
+struct RouterContext2: Hashable {
```

Comment:
There is no RouterContext inside code (at least didn't find any). So I guess you could use RouterContext as naming?

Reply:
Legacy Context xD
will delete